libautoscoper/src/VolumeDescription.hpp

```diff
@@ -68,6 +69,8 @@ class VolumeDescription
   const float* invScale() const { return invScale_; }
   const float* invTrans() const { return invTrans_; }
   const double* transCenter() const { return transCenter_; }
+  Vec3f transCenterVec() { return Vec3f(transCenter_); }
```

Comment:
Considering `transCenter_` is a vector of double, should this return `Vec3d` instead of `Vec3f` ?